src/components/Layouts/Navbar.jsx

```diff
@@ -46,7 +46,7 @@ function Navbar({ dark }) {
         </div>
         <div className="flex">
           <Link to="/">
-            <span className="uppercase font-bold">Cryptoniq</span>
+            <span className="uppercase font-bold">Crypto-Nested</span>
```

Comment:
- [ ] I am adding this here but it is for almost all your files, you disabled too many linters, which is not good because linters are here to guide you to writing clean codes, kindly minimize your linters disabled across your codebase